RELEASE/scripts/autoscend/paths/heavy_rains.ash

```diff
@@ -385,7 +385,7 @@ boolean rainManSummon(monster target, boolean copy, boolean wink)
 	}
 
 	
-	if(item_amount($item[Rain-Doh black box]) > 0 ||			//do we actually have a black box to copy with
+	if(item_amount($item[Rain-Doh black box]) < 0 ||			//do we actually have a black box to copy with
```

Comment:
Shouldn't this be == 0 or  < 1? It seems unlikely we'll have less than 0.

Reply:
Right you are.